pkg/op-brains/op_brains/summarizer/summarize_out_of_date_threads.py

```diff
@@ -0,0 +1,103 @@
+import os
```

Comment:
this file was inspired by `pkg/op-brains/op_brains/summarizer/summarize_all_threads.py`, the difference is that we take only the unsummarized topics, summarize them and then save the summaries in the database